samples/Shell/Fabimals/Fabimals/Components/SearchHandlers.fs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright 2018-2019 Fabulous contributors. See LICENSE.md for license.
+namespace Fabimals.Components
+
+open Fabulous.DynamicViews
+open Xamarin.Forms
+open Fabimals.Models
+
+module SearchHandlers =
+    type Msg =
+        | QueryChanged of string
+        | AnimalSelected of Animal
+
+    let animalSearchHandler animals dispatch =
+        View.SearchHandler(
+            placeholder="Enter search term",
+            showsResults=true,
+            queryChanged=(fun (_, newValue) -> dispatch (QueryChanged newValue)),
```

Comment:
Do you think we can debounce this? 
I mean that not on every change the message is run - just curious if it is possible.

Reply:
It is possible, yes. It's quite easy:
```fsharp
queryChanged=debounce 250 (fun (_, newValue) -> dispatch (QueryChanged newValue)),
```

I tried it with different times, but the UX doesn't feel satisfying.
The delay is the UI is not great.

Reply:
hmmm... does every message run or only the latest with the debounce ?
maybe if only the latest massage will run it could be a bit better.

Reply:
debounce only sends the last message in a given time frame.
Everytime the event is triggered, it will wait for 250ms to check if there's no other event triggers. Only then it really calls the function and dispatches.

So this delay is kinda bad when typing. It can give a few quirks like if you're typing too slow, it will show delayed results not matching what you're currently typing.
E.g. You're typing "ABCD" slowly. While you wrote "ABC", it will display results for "AB".

So it's confusing.

Reply:
well i think maybe playing a bit with the timeframe could be a good way. I mean RX or ReactiveUI uses nearly the same logic for throttle or sth. else. There it works great, so I thought this could be a great thing to have. 

Maybe we need to investigate on this and make mit more usable -> new issue?